app.js

```diff
@@ -2,22 +2,120 @@
   if (!window.addEventListener) return // Check for IE9+
 
   let options = INSTALL_OPTIONS
+  let updateTimeout
+  const delay = 1500
+  const elements = []
+  const PAYPAL_SCRIPT_URL = "https://cdn.rawgit.com/paypal/JavaScriptButtons/master/dist/button.js"
+  const TIME_PERIOD_SYMBOLS = {
+    D: "Daily",
+    W: "Weekly",
+    M: "Monthly",
+    Y: "Yearly"
+  }
+
+  function updateElements() {
+    if (options.merchant) {
+      const {buttons} = options
+
+      buttons
+        .filter($ => $.name && $.amount)
+        .forEach((attrs, i) => {
+          const paypalButton = document.createElement("script")
+          const paypalInfoWrapper = document.createElement("eager-paypal-info-wrapper")
+          const paypalItemName = document.createElement("eager-paypal-item-name")
+          const paypalPrice = document.createElement("eager-paypal-price")
+
```

Comment:
Move all the common button data attributes above the button type specific code.

Additionally, add the `data-env="sandbox"` attribute when in the using the app [previewer](https://eager.io/developer/docs/writing-your-app/environment).